<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpidrMonDialog</class>
 <widget class="QDialog" name="SpidrMonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>290</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>275</width>
    <height>290</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>275</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SPIDR-MON</string>
  </property>
  <property name="windowIcon">
   <iconset resource="spidrmon.qrc">
    <normaloff>:/icon/spidrmon.ico</normaloff>:/icon/spidrmon.ico</iconset>
  </property>
  <widget class="QGroupBox" name="groupBoxSpidr">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>4</y>
     <width>259</width>
     <height>95</height>
    </rect>
   </property>
   <property name="title">
    <string>SPIDR</string>
   </property>
   <widget class="QLabel" name="labelPeriod">
    <property name="geometry">
     <rect>
      <x>46</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr0">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr2">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>168</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="_pushButtonConnectOrDisconnect">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr1">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPeriod_3">
    <property name="geometry">
     <rect>
      <x>122</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelIpAddr">
    <property name="geometry">
     <rect>
      <x>58</x>
      <y>14</y>
      <width>69</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP address</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>14</y>
      <width>53</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelPeriod_2">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>34</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAddr3">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditPort">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>50000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="_labelDisconnected">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>64</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="text">
     <string>DISCONNECTED</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxMonitor">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>104</y>
     <width>259</width>
     <height>177</height>
    </rect>
   </property>
   <property name="title">
    <string>Monitoring info</string>
   </property>
   <widget class="QLineEdit" name="_lineEditRemoteTemp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>30</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditLocalTemp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>54</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAvddMvolt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>98</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAvddMamp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>98</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditAvddMwatt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>98</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelAvdd">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>100</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AVDD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelRemoteTemp">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>32</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remote</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLocalTemp">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>54</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMvolt">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>82</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[mV]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditDvddMwatt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>122</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditDvddMvolt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>122</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditDvddMamp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>122</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditVddMwatt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>146</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditVddMvolt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>146</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="_lineEditVddMamp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>146</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelRemoteTemp_2">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>14</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMvolt_2">
    <property name="geometry">
     <rect>
      <x>86</x>
      <y>82</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[mA]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelMvolt_3">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>82</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>[mW]</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelDvdd">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>124</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DVDD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelVdd">
    <property name="geometry">
     <rect>
      <x>206</x>
      <y>148</y>
      <width>43</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VDD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="_leUpdateLed">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>43</y>
      <width>25</width>
      <height>17</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>255</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>236</red>
          <green>233</green>
          <blue>216</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="spidrmon.qrc"/>
 </resources>
 <connections/>
</ui>
